aiohttp/web_urldispatcher.py

```diff
@@ -28,12 +29,25 @@
 __all__ = ('UrlDispatcher', 'UrlMappingMatchInfo',
            'AbstractResource', 'Resource', 'PlainResource', 'DynamicResource',
            'AbstractRoute', 'ResourceRoute',
-           'StaticResource', 'View')
+           'StaticResource', 'View', 'RoutesDef',
+           'head', 'get', 'post', 'patch', 'put', 'delete', 'route')
 
 HTTP_METHOD_RE = re.compile(r"^[0-9A-Za-z!#\$%&'\*\+\-\.\^_`\|~]+$")
 PATH_SEP = re.escape('/')
 
 
+class RouteDef(namedtuple('_RouteDef', 'method, path, handler, kwargs')):
+    # TODO: add __repr__
```

Comment:
As I said `Route` name  is captured by already existing class hierarchy.
It's present *instantiated route*.
In opposite *route definition* is for list of routes to be added to router instance.

I still think the distinction is matter.